services/slack/README.md

```diff
@@ -3,26 +3,96 @@ Slack
+
+### Naming Conventions
+
+#### Rule #1
+hyphens-not-underscores
 
-### ext-
+#### Rule #2
+all-lower-case
 
-Some channels begin with and ext-, these channels allow clients to join in on the conversation.
+#### Rule #3
+Including the year in a title name is generally unnecessary. This sets up a short-lived channels mindset. If you do find it necessary to set up one of those, it would be wise to go ahead and set up a calendar reminder to kill it when you likely know its usefulness would run out.
+
+#### Rule #4
+prefix-appropriately
+
+- `-` Important, common channels for everyone.
+The `-` ensures they go to the top of the list. A channel should get this prefix if everyone at Sparkbox would likely find value in belonging to it. *Example: `-biz`.*
+- `int-` Client Projects, "internal" Sparkboxer-only channels.
+- `ext-` Client Projects, "external" channels that may include the client team.
+- `temp-` Channels with a temporary purpose. You need something larger than a DM but it won’t be a permanent channel (example: conferences, Maker Series, etc.). Perhaps set up a calendar reminder to delete your channel later.
+
+### Pruning Channels
+
+Channels can quickly become unruly and difficult to navigate. We’ve recognized two reasons to archive a channel:
+
+- Redundancy: If two or more channels serve the same purpose, then decide on the one that is most popular/used, and delete the others.
+- Unused: If a channel has gone silent, then a vote will be called and requires three* people voting :hand: in order to keep the channel. If less than three people vote to keep the channel, then it should be archived.
+
+Unless a channel is redundant, the default action will be to archive it, because channels can be unarchived very easily. Since archived channels don’t shown in the channel listing, archiving is a safe way to remove visual clutter and cut down on searching while still leaving the option to resurrect that channel later.
+
+#### garbage_bot
+
+The bot user, @garbage_bot is a tool created by Patrick Simpson to identify unused channels on a quarterly basis. It will notify #random with a list of channels flagged for review.
+
+## Etiquette
+#### `@everyone`, `@channel`, or `@group`
+Try using `@here` to only notify online, desktop users. Alternatively, `@dayton` is useful to target only those in the Dayton office, or `@pittsburgh` for the Pittsburgh office. In any case, be respectful of the many time zones who will receive notifications, often to mobile devices on nightstands.
+
+#### Code of Conduct
```

Comment:
link to here: https://docs.google.com/document/d/1jrHZABYb2ZcYw6ZJ1CABODNwAxn_on-Hjga25Az73tg/edit#heading=h.p0yc4hw64s36

Reply:
let me ask Rob about that.